src/pkg/logger/logger.go

```diff
@@ -60,6 +61,7 @@ const (
 	LogLevelFN          = "log-level"
 	ReadableLogsFN      = "readable-logs"
 	MaskSensitiveDataFN = "mask-sensitive-data"
+	logStorageEngineFN  = "log-storage-engine"
```

Comment:
suggestion, since "storage-engine" sounds a bit flowery.
```suggestion
	logStorageFN  = "log-storage"
```